utils/src/main/resources/application.conf.template

```diff
@@ -18,3 +18,6 @@ enceladus.utils.testUtils.sparkTestBaseMaster = "local[4]"
 #enceladus.utils.testUtils.hadoop.conf.dir = "/path/to/HADOOP_CONF_DIR"
 #enceladus.utils.testUtils.spark.distJars.dir = "hdfs:///path/to/spark/jars"
 #enceladus.utils.testUtils.spark.home.dir = "/path/to/SPARK_HOME
+
+#system-wide time zone
+timezone="UTC"
```

Comment:
Missing newline

Reply:
Moot, as it is used for the Spark Session, which will be removed. So this change will be rolled back too.